stories/documentation/navigation/segmentedControl/segmentedControl-tabs.stories.ts

```diff
@@ -0,0 +1,92 @@
+import { Meta, StoryFn } from '@storybook/angular';
+
+interface segmentedControlTabsStory {
+	S: string;
+	withNumericBadge: boolean;
+	vertical: boolean;
+}
+
+export default {
+	title: 'Documentation/Navigation/segmentedControl/Tabs',
+	argTypes: {
+		S: {
+			control: {
+				type: 'boolean',
+			},
+		},
+		vertical: {
+			control: {
+				type: 'boolean',
+			},
+		},
+		tabActive: {
+			control: {
+				min: 1,
+				max: 4,
+				type: 'number',
+			},
+		},
+		withNumericBadge: {
+			control: {
+				type: 'boolean',
+			},
+		},
+	},
+} as Meta;
+
+function getTemplate(args: segmentedControlTabsStory): string {
+	const size = args.S ? `mod-S` : '';
+	const vertical = args.vertical ? `mod-vertical` : '';
+
+	return `<ul class="segmentedControl ${size} ${vertical}" role="tablist">
+	<li class="segmentedControl-item" role="presentation">
+		<button class="segmentedControl-item-action" type="button" role="tab" id="tab1" aria-controls="panel1" [attr.aria-selected]="tabActive === 1" [attr.tabindex]="tabActive === 1 ? null : '-1'">
```

Comment:
Isn't it [attr.tabindex]="tabActive === 1 ? 0 : '-1'"

Reply:
Both seem possible to me: a tabindex of 0 on a button is not necessary.